Assets/Scripts/ROS Scripts/ROS Connections v2/Drones/Matrice_ROSDroneConnection.cs

```diff
@@ -879,18 +874,20 @@ public void HandleUploadWaypointsTaskResponse(JSONNode response)
         response = response["values"];
         Debug.LogFormat("Waypoint task upload {0} (ACK: {1})", (response["result"].AsBool ? "succeeded" : "failed"), response["ack_data"].AsInt);
 
-        // Start flight upon completing upload
-        // Disabled for now
-        /*
+        // Start flight upon completing upload        
         if (response["result"].AsBool == true)
         {
-            SendWaypointAction(WaypointMissionAction.START);
+            Debug.Log("Executing mission");
+            
+            // @Eric,Nitzan: Uncomment as needed.
+            //SendWaypointAction(WaypointMissionAction.START);
         }
         else
         {
-            StartMission();
+            Debug.Log("Mission upload failed");
+            //StartMission();
```

Comment:
if mission upload failed, why do we still start mission?

Reply:
We don't 😅, will clean up that end point soon.